### DISCUSSION TITLE
有没有小伙伴讨论一下，只用pw2，不用adg等第三方情况下屏蔽广告的方法。

### DISCUSSION THREAD

**Discussion by sontjer** [Openwrt-Passwall/openwrt-passwall2#107] [General]
我只知道V2ray/Xray 分流规则，AD里，设置为 geosite:category-ads-all。其它有什么措施吗？这样设置了屏蔽广告，得分大概70多分，很多微信公众号的广告没法去掉。请大家畅所欲言。[测试网站:https://checkadblock.ru/](https://checkadblock.ru/)

**Comment by jh0072110** [comment]
广告也不是洪水猛兽…差不多去去就得了……

**Reply by sontjer** [reply]
好吧，我这里本意是讨论下技术层面，没想到要挡人饭碗。😃

**Comment by jh0072110** [comment]
不是挡饭碗…我以前也追求过一点广告都看不到的效果…但是…后面发现其实意义不大，因为不管是移动端还是电脑端都会习惯性的装去广告插件这是其一、第二是这玩意有点儿错杀一千不放过一个的感觉，就是要不杀了广告同时也杀掉了一部分正常信息，要不就是杀不干净，所以…我比较倾向于…打开就行了效果随它便的想法…

**Comment by Rashhz** [comment]
我也觉得屏蔽的七七八八就行了，所有的广告都要屏蔽肯定会误伤。到时候某些网页显示/功能不正常就得不偿失了

**Comment by martinzu** [comment]
大佬们，移动端有什么去广告的插件吗？感觉用了楼主的 geosite:category-ads-all 也没啥用。不如 chrome 装去广告插件有用
